backend/apid/graphql/service.go

```diff
@@ -83,8 +83,6 @@ func NewService(cfg ServiceConfig) (*graphql.Service, error) {
 	// Register user types
 	schema.RegisterRole(svc, &roleImpl{})
 	schema.RegisterRule(svc, &ruleImpl{})
```

Comment:
You'll need to register to `RoleBinding`, `ClusterRoleBinding`, `Subject`, `RoleRef`, and `ClusterRole` as well.

You should be able to use the generated field resolvers, eg.

```go
schema.RegisterRoleBinding(svc, &schema.RoleBindingAliases)
schema.RegisterClusterRoleBinding(svc, &schema.ClusterRoleBindingAliases)
// ...
```